common/common.cabal

```diff
@@ -13,6 +13,7 @@ library
                , containers
                , dependent-sum
                , dependent-sum-template
+               , fast-logger
```

Comment:
I'm a little puzzled about how this works because of:

https://github.com/kazu-yamamoto/logger/blob/master/fast-logger/fast-logger.cabal#L38

and

https://github.com/kazu-yamamoto/unix-time/blob/master/unix-time.cabal#L12

Any idea how much work it would take to copy the code and paste it into this repo?